steps/discover-repos.rb

```diff
@@ -51,6 +51,11 @@
 raise 'Can only retrieve up to 1000 repos' if opts[:total] > max
 
 size = [opts[:page_size], opts[:total]].min
+licenses_to_filter = [
+  { key: 'mit', name: 'MIT License' },
+  { key: 'apache-2.0', name: 'Apache License 2.0' },
+  { key: '0bsd', name: 'BSD Zero Clause License' }
```

Comment:
@Orillio why do we need their names? I believe, just `key` will be enough.

Reply:
In case someone wanted to include license names inside the exported files (csv/tex), they may become handy. 
I will remove them, if it causes issues.

Reply:
@Orillio it looks like information duplication at the moment